cpp/harvest/nacjd.cc

```diff
@@ -375,14 +456,34 @@ int Main(int argc, char *argv[]) {
     // Download possible new publications and store in json file
     LOG_INFO("Extracting ICPSR ids from website...");
     unsigned number_of_new_ids;
-    ExtractIDsFromWebsite(parsed_marc_ids, &number_of_new_ids, count_rows);
+    ExtractIDsFromWebsite(parsed_marc_ids, &number_of_new_ids, std::stoi(argv[4]));
```

Comment:
Since argv[4] is the optional `number`, we need to check for `argc == 5 ` and only pass the parameter in this case.